src/main/webapp/app/exercises/quiz/participate/quiz-participation.component.html

```diff
@@ -33,132 +34,110 @@ <h2>
+    <div class="row m-0 pb-5">
+        @if (quizExercise) {
+            <div class="col-md-auto">
+                <div class="p-0 col-md-auto quiz-navigation stepwizardquiz sticky-top">
+                    @for (question of quizExercise.quizQuestions; track question; let i = $index) {
+                        <div class="stepwizardquiz__step mb-3" [class.mb-3]="!$last">
+                            @if (question.type === DRAG_AND_DROP) {
+                                <span
+                                    class="btn btn-light btn-circle draganddropcolor-question"
+                                    (click)="navigateToQuestion(i)"
+                                    [ngbTooltip]="!!dragAndDropMappings.get(question.id!)?.length ? tooltipExplanationTranslate : tooltipNotExplanationTranslate"
+                                    [ngClass]="!!dragAndDropMappings.get(question.id!)?.length ? 'changed-question' : ''"
+                                >
+                                    <b class="fa">DD</b>
+                                </span>
+                            }
+                            @if (question.type === MULTIPLE_CHOICE) {
+                                <span
+                                    class="btn btn-light btn-circle multiplechoicecolor-question"
+                                    (click)="navigateToQuestion(i)"
+                                    [ngbTooltip]="!!selectedAnswerOptions.get(question.id!)?.length ? tooltipExplanationTranslate : tooltipNotExplanationTranslate"
+                                    [ngClass]="!!selectedAnswerOptions.get(question.id!)?.length ? 'changed-question' : ''"
+                                >
+                                    <b class="fa">MC</b>
+                                </span>
+                            }
+                            @if (question.type === SHORT_ANSWER) {
+                                <span
+                                    class="btn btn-light btn-circle shortanswercolor-question"
+                                    (click)="navigateToQuestion(i)"
+                                    [ngbTooltip]="!!shortAnswerSubmittedTexts.get(question.id!)?.length ? tooltipExplanationTranslate : tooltipNotExplanationTranslate"
+                                    [ngClass]="!!shortAnswerSubmittedTexts.get(question.id!)?.length ? 'changed-question' : ''"
+                                >
+                                    <b class="fa">SA</b>
+                                </span>
+                            }
+                            <ng-template #tooltipExplanationTranslate>{{ 'artemisApp.quizExercise.explanationAnswered' | artemisTranslate }} </ng-template>
+                            <ng-template #tooltipNotExplanationTranslate>{{ 'artemisApp.quizExercise.explanationNotAnswered' | artemisTranslate }} </ng-template>
+                        </div>
```

Comment:
(nitpick) Is it possible to use jhiTranslate with ng-template? If yes we should use it here to stay consistent with the coding guidelines.

Reply:
Should be possible 👍 (or at the very least, we could insert a `<span>` that has `jhiTranslate`)